ansible/roles_ocp_workloads/bookbag/tasks/remove_workload.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# Implement your Workload removal tasks here
```

Comment:
Nothing to remove?

Reply:
Oops! I missed pushing that commit! Pushed now. Just removing the namespace.